spec/lib/audit/catalog_to_moab_instance_spec.rb

```diff
@@ -109,6 +150,55 @@
         )
         c2m.check_catalog_version
       end
+
+      context 'check whether PreservedCopy already has a status other than OK_STATUS, re-check status if so' do
```

Comment:
similar to above comment  about condensing the written code, but not what is actually checked ...